src/services/__tests__/redisSessionStorage.test.ts

```diff
@@ -380,18 +376,18 @@ describe("RedisSessionStorage#set", () => {
       expect(mockSet.mock.calls[1][0]).toBe(
         `WALLET-${aValidUser.wallet_token}`
       );
-      expect(mockSet.mock.calls[1][1]).toBe(aValidUser.wallet_token);
+      expect(mockSet.mock.calls[1][1]).toBe(aValidUser.session_token);
```

Comment:
the use of calls[][] is a little bit confusing: maybe should be better something like
```
    expect(mockSet).toHaveBeenNthCalledWith(
        2,
        `WALLET-${aValidUser.wallet_token}`,
        aValidUser.session_token,
        expect.anything(),
        expect.anything(),
        expect.anything()
      );
```